lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -208,34 +252,30 @@ def verify_userauth(id_token, group_name=None, internal_token_only=False):
         keyset = None
         target_client_ids = None
 
-        if issuer == ROKWIRE_ISSUER:
```

Comment:
Hi @shurwit and @jsafoan, 

It looks like we missed this particular case when reviewing this PR. The Events Manager uses the Rokwire issuer and the tokens signed by it to talk to the Events Building Block without user login. So, at present, these are like service tokens used for nightly upload/update of events. Since this particular "if case" was removed in the PR, currently uploading WebTools events to Events Manager doesn't work in the DEV instance, affecting backward compatibility. Our team could fix it by bringing this back, but I wanted to get your thoughts before we do that. Thanks.

Reply:
Hi @sandeep-ps, sorry we missed that! Yes you are welcome to add this back in for now. At some point we will use the service accounts from the Core BB that we are discussing here https://github.com/rokwire/core-building-block/issues/306 to replace this, but for now it should be fine to continue supporting this mechanism. Feel free to make any changes needed to get this working again for now, and we can reconcile any issues when we discuss migrating this functionality to Core service accounts.

Reply:
No issues. We also missed it during the review. Will get this added.